### PR TITLE
docs(Form): supplement the documentation of Form

### PR DESCRIPTION
**Pull request by FanetheDivine**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution
补充`Form`组件文档中关于原生form属性的支持

### 📝 Changelog

- Use a developer-oriented tone and narrative style.
- Describe the user's first-hand experience of the issue and its impact on developers, rather than your solution approach.
- Refer to: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     supplement the documentation of `Form`      |
| 🇨🇳 Chinese |     补充`Form`组件文档      |

### CODE REVIEW

File: components/form/index.zh-CN.md [hunk 1]
```diff
@@ -93,6 +93,8 @@ coverDark: https://mdn.alipayobjects.com/huamei_7uahnr/afts/img/A*ylFATY6w-ygAAA
 | onValuesChange | 字段值更新时触发回调事件 | function(changedValues, allValues) | - |  |
 | clearOnDestroy | 当表单被卸载时清空表单值 | boolean | false | 5.18.0 |
 
+支持原生form除`onSubmit`外的所有属性。
```
Comment (afc163) [inline components/form/index.zh-CN.md:96]:
```suggestion
> 支持原生 form 除 `onSubmit` 外的所有属性。
```

File: components/form/index.en-US.md [hunk 2]
```diff
@@ -92,6 +92,8 @@ Common props ref：[Common props](/docs/react/common-props)
 | onValuesChange | Trigger when value updated | function(changedValues, allValues) | - |  |
 | clearOnDestroy | Clear form values when the form is uninstalled | boolean | false | 5.18.0 |
 
+It accepts all props which native forms support but `onSubmit`.
```
Comment (afc163) [inline components/form/index.en-US.md:95]:
```suggestion
> It accepts all props which native forms support but `onSubmit`.
```

**Review by afc163 — APPROVED** [review]

File: components/form/index.zh-CN.md [hunk 3]
```diff
@@ -93,6 +93,8 @@ coverDark: https://mdn.alipayobjects.com/huamei_7uahnr/afts/img/A*ylFATY6w-ygAAA
 | onValuesChange | 字段值更新时触发回调事件 | function(changedValues, allValues) | - |  |
 | clearOnDestroy | 当表单被卸载时清空表单值 | boolean | false | 5.18.0 |
 
+> 支持原生 form 除 `onSubmit` 外的所有属性。
```
Comment (afc163) [inline components/form/index.zh-CN.md:96]: 不对，onSubmit 应该也支持。

Comment (FanetheDivine) [reply components/form/index.zh-CN.md:96]: 我刚才试了一下是不行的,而且源码里有这么一段`type BaseFormProps = Omit<React.FormHTMLAttributes<HTMLFormElement>, 'onSubmit' | 'children'>;`直接把这两个属性排除了 children不是原生的没写

Comment (FanetheDivine) [reply components/form/index.zh-CN.md:96]: @afc163 我刚才试了一下是不支持的

屏幕截图 2024-08-04 001750
而且源码里有这么一段`type BaseFormProps = Omit<React.FormHTMLAttributes<HTMLFormElement>, 'onSubmit' | 'children'>;`直接把onSubmit排除了

**Review by afc163 — APPROVED** [review]

**Pull request merged**